tests/digraph/test_nodes.py

```diff
@@ -76,6 +76,12 @@ def test_remove_nodes_from_with_invalid_index(self):
         self.assertEqual(["a"], res)
         self.assertEqual([0], dag.node_indexes())
 
+    def test_has_node(self):
+        dag = rustworkx.PyDAG()
+        node_a = dag.add_node("a")
+        self.assertTrue(dag.has_node(node_a))
+        self.assertFalse(dag.has_node(node_a + 1))
```

Comment:
Can you add an equivalent test for `PyGraph` to `tests/graph/test_nodes.py`?

Reply:
done